README.md

```diff
@@ -1,8 +1,13 @@
-# Docker images for Kafka
+# Docker images for Apache Kafka
```

Comment:
@rmoff fyi, this file doesn't get uploaded to docker hub. just want to make sure you realize that.